### PR TITLE
Embed redesign 2/n: Sidebar sections + Sandbox info

### PR DESCRIPTION
**Pull request by siddharthkp**
_The goal of this PR is to move closer to the final design without making the experience worse. (`n + 1 > n`)_

Built on top of https://github.com/codesandbox/codesandbox-client/pull/2678

### Part 1

Changes the color of the shell without changing the components. The goal of this PR is to move closer to the final design without making the experience worse. (n + 1 > n)

Implementation detail: Created a new VSCode theme that matches our design

Using it in embed. Had to modify a few components (2) to use the right keys

### Part 2

Changes the structure of the sidebar by adding foldable sections + redesigned the sandbox info tab

Added stats to the sandbox info section

### Demo

https://pr2731.build.csb.dev/embed/5pj49

## What steps did you take to test this?

- [x] Testing embed applies new styles
- [x] Testing sandbox remains the same

## Checklist

- [NA] Documentation
- [x] Testing 
- [x] Ready to be merged 
- [NA] Added myself to contributors table

### CODE REVIEW

**Comment by siddharthkp** [comment]
@DannyRuchtie Quick question: 

The redesign also introduces stats on an embedded sandbox. Should clicking on the like/fork icon trigger an action?

(for comparison, the heart on app triggers a like, the fork doesn't do anything)

Screenshot 2019-10-14 at 11 44 25 AM

**Comment by DannyRuchtie** [comment]
It would be nice!

**Comment by siddharthkp** [comment]
> It would be nice!

- [ ] Good to have?

or

- [ ] It's silly to show the icons without an action because it's breaks expectation

**Comment by DannyRuchtie** [comment]
Let's say it's nice to have.
Not prio #1 but a really nice to have! 
currently, we don't have the actions (explore, and viewing someone's sandbox)
 
Schermafbeelding 2019-10-14 om 11 51 58
Schermafbeelding 2019-10-14 om 11 51 47

**Comment by siddharthkp** [comment]
@DannyRuchtie Does this look like a good n+1 step to ship?

**Comment by CompuIves** [comment]
This looks awesome!! A couple things I'd like some discussion on:

1. Do we want to open the files item in the sidebar by default? 
2. Does it make sense to have something on hover for the file tabs (in the editor)?

File: packages/app/src/embed/components/Content/elements.ts [hunk 1]
```diff
@@ -3,15 +3,15 @@ import styled from 'styled-components';
...
+  background-color: ${props => props.theme['editor.background']};
   height: calc(100% - 2.5rem);
 `;
 
 export const Tabs = styled.div`
   display: flex;
   height: 35px;
   min-height: 35px;
-  background-color: rgba(0, 0, 0, 0.3);
+  background-color: ${props => props.theme['tab.inactiveBackground']};
```
Comment (CompuIves) [inline packages/app/src/embed/components/Content/elements.ts]: Nice that we're using themes here already!

**Comment by DannyRuchtie** [comment]
1) Do we want to open the files item in the sidebar by default?
Makes sense. But because of the big open in Codesandbox button and logo. It currently looks a bit cluttered. 

2) Does it make sense to have something on hover for the file tabs (in the editor)?
I agree animate font color to white would be nice 

Two things: 
1) The tabbar height seems to be 35px (should be aligned with the sidebar headers)
Schermafbeelding 2019-10-15 om 16 39 09

2) Font size of the name is 12px (was 11px to small?)

**Comment by siddharthkp** [comment]
Superseded by #2791

**Pull request closed without merging**